lib/mergometer.rb

```diff
@@ -3,19 +3,18 @@
 
 require "mergometer/version"
 require "mergometer/configuration"
-
-reports_path = File.expand_path("../mergometer/reports/**/*.rb", __FILE__)
-Dir[reports_path].each { |f| require f }
```

Comment:
(This change seems unrelated? Maybe not necessary? `Report` itself does not depend on the individual reports)

Reply:
Yeah I guess (it does later though :P). Will remove this.